_includes/payment-order-other-features.md

```diff
@@ -601,17 +601,48 @@ Content-Type: application/json
+| {% icon check %} | └➔&nbsp;`amount`               | `integer`    | {% include field-description-amount.md %}                                                                                                                                                                                                                                             |
+| {% icon check %} | └➔&nbsp;`vatAmount`            | `integer`    | {% include field-description-vatamount.md %}                                                                                                                                                                                                                                          |
+| {% icon check %} | └➔&nbsp;`description`          | `string`    | {{% include field-description-description.md documentation_section="paymentorders" %}                                                                                                                                                                                                                                           |
+| {% icon check %} | └─➔&nbsp;`userAgent`           | `string`     | The user agent reference of the consumer's browser - [see user agent definition][user-agent-definition]                                                                                                                                                                                                                                                         |
+| {% icon check %} | └─➔&nbsp;`language`            | `string`     | {% include field-description-language.md api_resource="paymentorders" %}                                                                                                                                                                                                                                           |
+| {% icon check %} | └─➔&nbsp;`urls`                | `string`     | {he URI to the `urls` resource where all URIs related to the payment order can be retrieved.                                                                                                                                                                                                                                           |
+| {% icon check %} | └─➔&nbsp;`callbackUrl`         | `string`      | The URL that Swedbank Pay will perform an HTTP `POST` against every time a transaction is created on the payment. See [callback][callback] for details.                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5741678)